sites/footers/reports/a11y-1.json

```diff
@@ -0,0 +1,905 @@
+      "earl:test": "WCAG20:captions-live",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G78",
+        "WCAG21Tech:general/G173",
+        "WCAG21Tech:general/G8",
+        "WCAG21Tech:general/G203"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.3.1 Info and Relationships (Level A)",
+      "earl:test": "WCAG20:info-and-relationships",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G115",
+        "WCAG21Tech:general/G117",
+        "WCAG21Tech:general/G140",
+        "WCAG21Tech:failures/F2",
+        "WCAG21Tech:failures/F33",
+        "WCAG21Tech:failures/F34",
+        "WCAG21Tech:failures/F42",
+        "WCAG21Tech:failures/F43",
+        "WCAG21Tech:failures/F46",
+        "WCAG21Tech:failures/F48",
+        "WCAG21Tech:failures/F87",
+        "WCAG21Tech:failures/F90",
+        "WCAG21Tech:failures/F91",
+        "WCAG21Tech:failures/F92"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.3.2 Meaningful Sequence (Level A)",
+      "earl:test": "WCAG20:meaningful-sequence",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G57",
+        "WCAG21Tech:html/H34",
+        "WCAG21Tech:html/H56",
+        "WCAG21Tech:css/C6",
+        "WCAG21Tech:css/C8",
+        "WCAG21Tech:css/C27",
+        "WCAG21Tech:failures/F1",
+        "WCAG21Tech:failures/F32",
+        "WCAG21Tech:failures/F33",
+        "WCAG21Tech:failures/F34",
+        "WCAG21Tech:failures/F49"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.3.3 Sensory Characteristics (Level A)",
+      "earl:test": "WCAG20:sensory-characteristics",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G96",
+        "WCAG21Tech:failures/F14",
+        "WCAG21Tech:failures/F26"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.3.4 Orientation (Level AA)",
+      "earl:test": "WCAG21:orientation",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G214",
+        "WCAG21Tech:failures/F97",
+        "WCAG21Tech:failures/F100"
+      ],
+      "earl:mode": "earl:semiAuto",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.3.5 Identify Input Purpose (Level AA)",
+      "earl:test": "WCAG21:identify-input-purpose",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
```

Comment:
There is no input control in the site footer, this is probably not applicable.

Reply:
As discussed with Brahim, this is what the automated tool has reported. Let's keep it as is.